core-java-modules/core-java-21/src/test/java/com/baeldung/java21/SwitchPatternTest.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.java21;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class SwitchPatternTest {
+
+    @Test
+    void whenUsingDifferentSwitchPatternThenReturnTheSameResult() {
```

Comment:
Saajan makes reference to test methods like this.

So the test method name should be: givenXXXX_whenYYY_thenZZZ